modules/library/render/src/test/java/org/geotools/renderer/lite/LetterConflictTest.java

```diff
@@ -55,8 +55,8 @@
      *
      * <p>Build with mvn -P image.interactive
      */
-    static final boolean IMAGE_INTERACTIVE =
-            Boolean.getBoolean("org.geotools.image.test.interactive");
+    static final boolean IMAGE_INTERACTIVE = true;
+    // Boolean.getBoolean("org.geotools.image.test.interactive");
```

Comment:
This should be restored to its prior state before merging

Reply:
Yep